tensorflow/buildspec-2-8.yml

```diff
@@ -6,12 +6,12 @@ short_version: &SHORT_VERSION 2.8
 arch_type: x86
 
 repository_info:
-  training_repository: &TRAINING_REPOSITORY
-    image_type: &TRAINING_IMAGE_TYPE training
-    root: !join [ *FRAMEWORK, "/", *TRAINING_IMAGE_TYPE ]
-    repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *TRAINING_IMAGE_TYPE]
-    repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/,
-      *REPOSITORY_NAME ]
+  # training_repository: &TRAINING_REPOSITORY
```

Comment:
We do not need to comment this section. Commenting the training portions under [images](https://github.com/aws/deep-learning-containers/pull/2538/files#diff-3af2656db81add8a43df467b23e486c7373b1034eed776f4fbb409c1353b5ce2L43) is enough. Please revert this change before merging.

Reply:
Arjuna would say differently - to comment it out.

Reply:
Commenting these sections when not required might lead to delays since we would need to get reviews from the respective code owners.